### PR TITLE
separating_gut_meosthelial_from_gut_lining

### PR DESCRIPTION
**Pull request by dosumis**
- Mesothelial cells of gut now part of serosa
- tidying up partonomy for intestinal lining
- Fixed brush border on enteroycte

Fixes https://github.com/obophenotype/cell-ontology/pull/2674 https://github.com/obophenotype/cell-ontology/issues/2825

### CODE REVIEW

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by dosumis** [comment]
Diffs are a bit odd.  Maybe some edits bleeding in from Master?

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
@aleixpuigb @dosumis This PR is ready for review, and all conflicts have been resolved. I am not sure why gogoeditdiff is not showing the latest updates (ie. this should be **EquivalentTo**`'mesothelial cell' and ('part of' some 'serosa of small intestine') `  and **SubclassOf** `'part of' some 'epithelium of small intestine')`

image

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
Some additional changes were made to this PR during my meeting with @dosumis this morning.

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Some inferences gave been lost:
- early colonocyte -> colonocyte

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -24559,8 +24563,7 @@ AnnotationAssertion(owl:deprecated obo:CL_0011107 "true"^^xsd:boolean)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:NV") obo:IAO_0000115 obo:CL_0011108 "Epithelial cell that is part of the colon epithelium.")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0011108 "colonic epithelial cell")
 AnnotationAssertion(rdfs:label obo:CL_0011108 "colon epithelial cell")
-EquivalentClasses(obo:CL_0011108 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001155)))
-SubClassOf(obo:CL_0011108 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000397))
+EquivalentClasses(obo:CL_0011108 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000397)))
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:24566]: Fixed

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**